dj_cqrs/transport/rabbit_mq.py

```diff
@@ -66,7 +66,7 @@ def produce(cls, payload):
     def _consume_message(cls, ch, method, properties, body):
         try:
             dct = ujson.loads(body)
-            for key in ('signal_type', 'cqrs_id', 'instance_data'):
+            for key in ('signal_type', 'cqrs_id', 'instance_data', 'tracked_fields_data'):
```

Comment:
The more I think on that, the less I see the reasons to break backward compatibility. Consider the case, when there will be no downtime for upgrade, so there may be loses of important updates.

I guess, the support for that field should be optional by nature. If you don't want to deal with changes, you could have the transport that just ignores them, for example.